qemu/tests/cfg/drive_mirror_negative_tests.cfg

```diff
@@ -53,13 +53,16 @@
             start_firewall_cmd = "iptables -A INPUT -p tcp -d 127.0.0.1 --dport 2049 -j DROP"
         - on_iscsi:
             image_format_target = raw
+            only raw
+            only 2raw
```

Comment:
-2raw define "image_format_target = "raw"" too,  so no need to add "image_format_target = raw" again when set "only 2raw".